robottelo/cli/base.py

```diff
@@ -86,6 +86,33 @@ class Base(object):
     logger = logging.getLogger('robottelo')
 
     @classmethod
+    def _handle_response(cls, response, ignore_stderr=None):
+        """Verifies ``return_code`` and ``stderr`` in ``SSHCommandResult``
+        response and throws ``CLICommandExecutionError`` exception if
+        ``return_code`` is not equal to 0.
+
+        """
+        if response.return_code != 0:
+            raise CLICommandExecutionError(
+                response.return_code,
+                response.stderr,
+                'Command \'{0} {1}\' finished with return_code {2}\n'
+                'stderr contains following message:\n{3}'
+                .format(
+                    cls.command_base,
+                    cls.command_sub,
+                    response.return_code,
+                    response.stderr,
+                )
+            )
+        if len(response.stderr) != 0 and not ignore_stderr:
+            cls.logger.warning(
```

Comment:
This should be discussed. I don't know whole backstory, but AFAIK, @elyezer once tried to count on `stderr` and fail tests when `stderr` is not empty. Then it appeared that some commands return some garbage in `stderr` and `remove_task_status()` was created for such commands. But it looks like in the end this approach was totally rejected as now we don't check for `stderr` in the latest test. Is that so?
Depending on the answer, I can see 3 approaches we can use for `stderr`:

1) (Currently applied) Leave `remove_task_status()` for commands as it was for compatibility, in case of `len(stderr) != 0` just throw warning in logs.
2) Remove `remove_task_status()` completely, do not check `stderr` at all
3) Count on `stderr`. Commands which produce some garbage in `stderr` should use `remove_task_status` flag, if flag is not used and `stderr` is not empty - throw an exception. The problem here is that we haven't covered all the commands with checks for `len(stderr)`, so theoretically we can face a lot of new failures. On the other hand, literally all the CLI tests should be refactored because of this PR, so we'll be able to catch all the failures and fix them.